ios/xcode/MGLKit/MGLKView.mm

```diff
@@ -89,12 +89,10 @@ - (void)setContext:(MGLContext *)context
     _context = context;
 }
 
-- (void)setEnableSetNeedsDisplay:(BOOL)value
+- (void)setNeedsDisplay
 {
-    if (value)
-    {
-        Throw(@"setNeedsDisplay not implemented");
-    }
+    if (_enableSetNeedsDisplay)
```

Comment:
Could you intialize this variable to YES in constructor. It matches the default value of GLKView’s.
Also guard this method inside `if !TARGET_OS_OSX` since macOS doesn’t support setNeedsDisplay method.
Thanks,

Reply:
Updated as requested.